client/orchestrator/client_orchestrator.go

```diff
@@ -58,6 +59,20 @@ type ProvisionDeviceReq struct {
 	Device model.Device `json:"device"`
 }
 
+// UpdateDeviceStatusReq contains request data of request to start update
+// device status  workflow
+type UpdateDeviceStatusReq struct {
+	// Request ID
+	RequestId string `json:"request_id"`
+	// Device IDs
+	//Ids []string `json:"device_ids"`
```

Comment:
which is it then? what's up with the list of strings?

Reply:
see below, I will remove the commented line.